support/ruby/collins-client/lib/collins/api/asset.rb

```diff
@@ -123,6 +123,40 @@ def find_similar asset_or_tag, size = 50, sort = "ASC", sort_type = "distance",
       end
     end
 
+    # Count number of assets matching the specified criteria
+    #
+    # @param [Hash] options Query options (same as in the "find" method)
+    # @return integer The number of assets matching the query
+    # @raise [UnexpectedResponseError] If the HTTP response code is not a 200
+    def count options = {}
+      # create a copy so that we do not modify the original options array
+      options = options.dup
+
+      if options.include? "size" or options.include? "page"
```

Comment:
does options hash automatically convert symbol keys to strings for this ```#include?``` operator?